src/invoices/InvoicesList/InvoiceLine/InvoiceLineEditLayer.js

```diff
@@ -24,6 +25,7 @@ class InvoiceLineEditLayer extends Component {
   static manifest = Object.freeze({
     invoice: invoiceResource,
     invoiceLine: invoiceLineResource,
+    poLine: poLineResource,
```

Comment:
I don't think we need this on the Edit form, in the ticket I found only Invoice line View requirement. I think you can just remove it from the form at all.